src/main/java/com/marklogic/appdeployer/AppConfig.java

```diff
@@ -199,6 +199,12 @@
 	private Pattern resourceFilenamesExcludePattern;
 	private Pattern resourceFilenamesIncludePattern;
 
+	//Fields to exclude from JSON payloads
+	private String[] excludeFields;
```

Comment:
I just checked the Manage docs, and I chose the wrong term - it should be "excludeProperties" instead of "excludeFields". Sorry about that!